app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,96 +221,173 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
+To be eligible for the second instalment of the IRP, you must remain on an eligible visa throughout your employment in England. 
+You can change your visa, as long as any new visa appears on the list of eligible visas on this page (example 1). You will not need to apply for any new visa from outside the UK.
+If your new visa is not listed on this page, you will not be eligible for the second instalment (see example 2).
+
+#### Example 1
+You arrive in the UK in September 2024 on a Youth Mobility Scheme visa. You are employed as a teacher on a contract lasting 1 academic year, and successfully apply for the first instalment of the IRP. 
+
+At the end of the academic year, your school offers to make your contract permanent and sponsor you for a Skilled Worker visa for 3 years. As your Youth Mobility Scheme visa is only valid for 2 years, and you are planning to work as a teacher in England for longer than this, you agree to switch your visa. 
+
+You apply for the second instalment of the IRP in September 2025. You remain eligible because the Youth Mobility Scheme visa and the Skilled Worker visa are on the list of eligible visas on this page. 
+
+#### Example 2
+You arrive in the UK in September 2024 on a Youth Mobility Scheme visa. You are employed as a teacher on a contract lasting 2 academic years and successfully apply for the first instalment of the IRP. 
+
+Early in 2025, your partner moves to the UK on a Skilled Worker visa lasting 3 years. You visit family overseas during the summer holidays. As your Youth Mobility Scheme visa is limited to 2 years, you decide to apply to join your partner as a dependant on their visa instead. You return to the UK to continue your employment in September 2025 on a dependant visa.
+
+You are not eligible for the second instalment of the IRP as the dependant visa does not appear on the list of eligible visas on this page.
+
+
+
+## How to apply for the IRP
 
-### Payment
+We will update this page with a link to the application form when it is live.
 
-The international relocation payment (IRP) cannot be paid to you in advance, before you take up your teaching job. This is because eligibility requirement checks can only be completed after you’ve arrived in England.
+Before you apply for the first and second instalments of the IRP, you’ll need to check you meet the eligibility requirements.
 
-In order to be paid the IRP, you must have a UK bank account in your name (this can be digital). The IRP cannot be paid into a building society account.
+Applications are open from 30 September 2024 to end of March 2025.
+
+If you start your teaching job between 1 March 2024 and 28 February 2025 you should apply between 30 September 2024 and the end of March 2025. 
+
+If you are eligible, you should receive the money following successful completion of our eligibility checks. You must apply between 30 September 2024 and the end of March 2025 to remain eligible.
+
+
```

Comment:
### Future application windows for IRPs

Applications will open again from 30 September 2025 to end of March 2026.  

If you start your teaching job between 1 April 2025 and 31 May 2025, you can apply when applications re-open in September 2025. 

If you start your teaching job in March 2025, you can either apply before applications close at the end of March 2025 or when they open again in September 2025.   

You can apply for IRP instalment 2 once you have completed 3 full terms teaching. 

DfE cannot process late applications for the IRP. If you do not apply during the correct time period, DfE will not process your application.